arviz/plots/pairplot.py

```diff
@@ -131,13 +131,20 @@ def plot_pair(
 
     if plot_kwargs is None:
         plot_kwargs = {}
-
-    if kind == "scatter":
-        plot_kwargs.setdefault("marker", ".")
-        plot_kwargs.setdefault("lw", 0)
+    else:
+        if kind == "scatter":
+            plot_kwargs = matplotlib_kwarg_dealiaser(plot_kwargs, "scatter")
+            plot_kwargs.setdefault("marker", ".")
+            plot_kwargs.setdefault("lw", 0)
```

Comment:
I'd use linewidth here. For now it works, but I have tried it and it will raise an error in the next release:

```
plt.plot([1,2], [2,0], ls="--", linestyle=":")                                                                                                                                                      
/home/oriol/venvs/arviz-dev/bin/ipython:1: MatplotlibDeprecationWarning: Saw kwargs ['ls', 'linestyle'] which are all aliases for 'linestyle'.  Kept value from 'linestyle'.  Passing multiple aliases for the same property will raise a TypeError in 3.3.
```

Reply:
Okay, I'll do the changes.